pac_man/simple-path.txt

```diff
@@ -0,0 +1,6 @@
+11111111111
+10000100001
+11111100001
+10000100001
+10000100001
+11111111111
```

Comment:
It is usually better to use underscores for filenames.
A maybe the filename could include "board" to be more explicit.